gluten-ut/src/test/scala/io/glutenproject/utils/clickhouse/ClickHouseNotSupport.scala

```diff
@@ -18,11 +18,50 @@
 package io.glutenproject.utils.clickhouse
 
 import io.glutenproject.utils.NotSupport
+
 import org.apache.spark.sql.catalyst.expressions._
 
 object ClickHouseNotSupport extends NotSupport {
 
-  override lazy val notSupportSuiteList: Map[String, Map[String, ExpressionInfo]] = Map.empty
+  override lazy val partialSupportSuiteList: Map[String, Seq[String]] = Map(
```

Comment:
Does an empty Seq mean none of test in this suite is supported?

Reply:
yes, all of them will be ignored